vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -570,15 +570,19 @@ supported:
           </Collection>
         </Annotation>
         <Property Name="Searchable" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
-          <Annotation Term="Core.Description" String="$search is supported" />
+          <Annotation Term="Core.Description" String="$search is supported with the standard syntax [OData-URL, section 5.1.8.1](https://docs.oasis-open.org/odata/odata/v4.02/odata-v4.02-part2-url-conventions.html#SearchExpressions)" />
         </Property>
         <Property Name="UnsupportedExpressions" Type="Capabilities.SearchExpressions" Nullable="false" DefaultValue="none">
           <Annotation Term="Core.Description" String="Expressions not supported in $search" />
         </Property>
+        <Property Name="SearchSyntax" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.IsURL" />
+          <Annotation Term="Core.Description" String="URL of the $search syntax supported by the service if this deviates from the standard syntax [OData-URL, section 5.1.8.1](https://docs.oasis-open.org/odata/odata/v4.02/odata-v4.02-part2-url-conventions.html#SearchExpressions)" />
+        </Property>
       </ComplexType>
       <EnumType Name="SearchExpressions" IsFlags="true">

```

Comment:
Can we add 
~~~xml
        <Member Name="all" Value="31">
          <Annotation Term="Core.Description" String="Only plain search terms are supported" />
        </Member>
~~~

Reply:
We can, but that does not mean that NOT may be interpreted as a term to be matched. See https://github.com/oasis-tcs/odata-specs/pull/1918/files#r1570163982